### ISSUE TITLE
Suggestion: Provide a hint about Menril

### ISSUE THREAD

**Issue by neoc666** [AllTheMods/ATM-Illumination#20]
I was desperately looking for Menril for quite some time. First I thought it doesn't even grow in the Overworld, since I once planted a sapling in Sky Factory 3 and it didn't grow. Later I read that there is a Menril biome, and so I didn't look very closely into normal biomes. Finally found one in Roofed Forest. A hint of some kind would have been nice.

**Comment by CDArena** [comment]
The Menril trees can be spotted at night - they are the blue trees that appear to glow a bit. I found one on an Extreme Hills hillside while waiting out the night. I ran across many of them while hunting for a jungle, so they are maybe "uncommon", but not rare. Also, Tech Reborn rubber trees look a bit different than IC2 versions - the new texture is more of a lighter grey fir - but the distinctive double leaf spike on top is still there, as are the rubber sap leaks.
